_data/investing.yml

```diff
@@ -177,7 +177,9 @@ websites:
       img: vanguard.png
       tfa: Yes
       sms: Yes
-      doc: https://personal.vanguard.com/us/insights/article/Account-security-092015
+      phone: Yes
+      hardware: Yes
```

Comment:
I don't see anything about hardware on the [doc](https://personal.vanguard.com/us/insights/article/security-codes-112015) page. Do they support it?